doc/manual/src/release-notes/rl-2.20.md

```diff
@@ -200,3 +200,8 @@
   while performing various operations (including `nix develop`, `nix flake
   update`, and so on). With several fixes to Nix's signal handlers, Nix
   commands will now exit quickly after Ctrl-C is pressed.
+
+- `nix copy` to a `ssh-ng` store now needs `--substitute-on-destination` (a.k.a. `-s`)
+  in order to substitute paths on the remote store instead of copying them.
+  The behavior is consistent with `nix copy` to a different kind of remote store.
+  Previously this behavior was controlled by `--builders-use-substitutes`.
```

Comment:
Was it?

Reply:
@Ericson2314 yes and I noticed because I relied on this behavior ;)
See also #10182

Reply:
That issues seems to confirms what I thought, which is `builders-use-substitutes` only affects building, and not manual `nix copy` usage!

Reply:
```suggestion
  Previously this behavior was controlled by the `builders-use-substitutes` and `--substitute-on-destination` was incorrectly ignored.
```

Reply:
-> #10458.